demisto_sdk/commands/common/content/objects/pack_objects/modeling_rule/modeling_rule.py

```diff
@@ -211,6 +211,12 @@ def __init__(self, path: Union[Path, str]):
         self._rules: List[SingleModelingRule] = []
         self.rules_dict: dict = {}
 
+    def __repr__(self) -> str:
+        path_name = (
+            self.path.name if isinstance(self.path, Path) else Path(self.path).name
+        )
```

Comment:
```suggestion
        path_name = Path(self.path).name
```

will work for both str / Path

Reply:
** Path** takes in a string, I wrote it as:
**path_name = Path(f'{self.path}').name**

Reply:
Path can take also a Path object if i am correct, check it